libraries/aleth/libdevcore/CommonJS.h

```diff
@@ -92,4 +92,12 @@ inline uint64_t jsToInt(std::string const& _s) { return std::stoull(_s, 0, 16);
 
 inline std::string jsToDecimal(std::string const& _s) { return toString(jsToU256(_s)); }
 
+inline uint64_t getUInt(auto const& v) {
```

Comment:
why not simply const std::string& if you dont call with with any other argument type ?

Reply:
this function taking `Json::Value` argument, not `std::string`

Reply:
Specifying it explicitly is better, but we will need to link jsoncpp library to libdevcore.

Reply:
if those fields that you are parsing by this are mandatory (which probably should be), you would not even need this function - problem solved

Reply:
Maybe you didnt get what this function do. It is not about mandatory field or not. It is about the field type. If we passing int as string (for example `"52"`, but not just `52`), we should parse it different ways 
This code was in several places, so I just moved it to the function

Reply:
oh yeah I did not read the code properly and mistakenly translated `return v.asUInt64();` as returning some default uint64 value, my bad